.gitignore

```diff
@@ -10,3 +10,4 @@ qmcpack_manual.pdf
 /build_*/
 docs/_build
 nexus/tests/unit/*output
+tests/solids/NiO_a4_e48_pp/NiO-fcc-supertwist111-supershift000-S1.h5
```

Comment:
I have this file locally as well.

Reply:
All the tests in NiO_a4_e48_pp handle this file specially. The file ought to be linked into the cmake build+test directories. I think dirtying the git repo is bad practice, so this is just a fast workaround to hide it.